frontend/pages/topics/index.tsx

```diff
@@ -24,16 +53,11 @@ const Topics: FC<TopicsProps> = ({ topicsData }: TopicsProps) => {
         <div style={{ marginBottom: "2rem", textAlign: "center" }}>
           {topics.map((topic) => (
             <Link href={`/topics/${topic.slug}`} key={topic.name} passHref>
-              <Button
-                component="a"
-                sx={{
-                  color: "black",
-                  margin: "0.6rem 1.2rem",
-                  display: "inline-block",
-                }}
-              >
-                {topic.name}
-              </Button>
+              <StyledTopicCard className={`m-2 ${className || ""}`} data-type={topic.model}>
+                <CardContent>
+                  <h3>{topic.name}</h3>
+                </CardContent>
+              </StyledTopicCard>
```

Comment:
I'm not sure this is an improvement over the previous layout. The cards are pretty bulky now, and the previous buttons had some nice hover effects that are removed here. Maybe `<Card/>` is not the best type of component for this page?

Do you plan to refine the design here more? I don't think this should be merged as-is.

Reply:
I see what you mean. I tried to make it easier to find a topic that you are looking for. The current Topics page is rather filled and hard to find what you want. Maybe a search implementation on the page would work well? What do you think @mattfredericksen ?

Reply:
I agree, the current page also has problems...

What do you think about adding dividers between each letter? I think it would be _an_ improvement, but I'm not sure if it's what you're looking for.

I don't have any other ideas off the top of my head, but typically its possible to find UI design examples for similar pages/features on other sites, which takes less effort and is usually higher quality than designs we could create ourselves, IMO. I would just run ideas by the team and/or Jacob before implementing anything non-trivial.